lib/token_authenticate_me/concerns/controllers/password_resetable.rb

```diff
@@ -18,13 +18,17 @@ module PasswordResetable
           def create
             @user = User.find_by_email(params[:email])
 
-            if @user
-              send_valid_reset_email(@user)
+            if (/@/ =~ params[:email]) == nil
```

Comment:
There is more requirements for an e-mail address, but it is not worth adding unless there is a concern.
From Wikipedia:
```
Invalid email addresses
Abc.example.com (no @ character)
A@b@c@example.com (only one @ is allowed outside quotation marks)
a"b(c)d,e:f;g<h>i[j\k]l@example.com (none of the special characters in this local-part are allowed outside quotation marks)
just"not"right@example.com (quoted strings must be dot separated or the only element making up the local-part)
this is"not\allowed@example.com (spaces, quotes, and backslashes may only exist when within quoted strings and preceded by a backslash)
this\ still\"not\\allowed@example.com (even if escaped (preceded by a backslash), spaces, quotes, and backslashes must still be contained by quotes)
1234567890123456789012345678901234567890123456789012345678901234+x@example.com (too long)
john..doe@example.com (double dot before @)
example@localhost (sent from localhost)
with caveat: Gmail lets this through, Email address#Local-part the dots altogether
john.doe@example..com (double dot after @)
a valid address with a leading space
a valid address with a trailing space
```